tailwind.config.js

```diff
@@ -1,16 +1,20 @@
 import { isolateInsideOfContainer, scopedPreflightStyles } from 'tailwindcss-scoped-preflight';
-import  form  from '@tailwindcss/forms';
+import form from '@tailwindcss/forms';
 
 export default {
   content: ['./src/**/*.{html,ts}'],
   theme: {
-    extend: {},
+    extend: {
+      fontFamily: {
+        sans: ['objektiv-mk1', 'sans-serif'],
```

Comment:
I based this one the fontFamily settings on [scottlogic.com](https://www.scottlogic.com/)